lib/util/dom.js

```diff
@@ -186,3 +192,74 @@ function findNodeInParents(node, targetTagName) {
   if (node.tagName.match(RegExp(targetTagName, 'i'))) return node;
   return findNodeInParents(node.parentNode, targetTagName);
 }
+
+/**
+ * Utility function to create a hidden input element
+ * @return {InputElement}
+ */
+function createHiddenInput (attributes = {}) {
+  let hidden = global.document.createElement(attributes.type === 'button' ? 'button' : 'input');
+  if (!('type' in attributes)) attributes.type = 'text';
+  if (!('style' in attributes)) attributes.style = 'position: absolute; top: 0px; left: -1000px; opacity: 0;';
+
+  Object.keys(attributes).forEach(attr => hidden.setAttribute(attr, attributes[attr]));
+
+  return hidden;
+}
+
+/**
+ * Utility function to determine if an input is intended to be hidden
+ * @return {Boolean}
+ */
+
+function isHiddenInput (element = undefined) {
+  return element !== undefined && element.style !== undefined && element.style.opacity === '0';
+}
+
+/**
+ * Determine if user's browser is mobile or non-mobile
+ * @return {Boolean} value
+ */
+
+function isMobile() {
+  return bowser.mobile || bowser.tablet;
+}
```

Comment:
This doesn't really fit the scope of the dom util since it doesn't interact with the dom.